File: features/wx/wx/Token.go
package wx

import (
	"github.com/hailongz/golang/db"
)

type Token struct {
	db.Object
	Type	int32	`json:"type" name:"type" title:"类型" index:"ASC"`
	Appid	string	`json:"appid" name:"appid" title:"appid" length:"64" index:"ASC"`
	AccessToken	string	`json:"-" name:"access_token" title:"access_token" length:"255"`
	Etime	int64	`json:"etime" name:"etime" title:"过期时间"`
}

func (O *Token) GetName() string {
	return "token"
}

func (O *Token) GetTitle() string {
	return "Token"
}

File: features/wx/ts/Token.ts
import { int64 } from "./lib/less";
import { UserType } from './UserType';

/**
 * Token
 * @type db
 */
export class Token {

    /**
     * ID
     */
    id: int64 = 0

    /**
     * 类型
     * @index ASC
     */
    type: UserType = UserType.MP

    /**
     * appid
     * @index ASC
     * @length 64
     */
    appid: string = ''

    /**
     * access_token
     * @length 255
     * @output false
     */
    access_token: string = ''

    /**
     * 过期时间
     */
    etime: int64 = 0

}